debian/opx-cps.postinst

```diff
@@ -9,8 +9,8 @@ if ! getent passwd  _opx_cps> /dev/null; then
     adduser --quiet --system  --force-badname --no-create-home --ingroup _opx_cps _opx_cps
 fi
 
-systemctl unmask redis-server
-systemctl start redis-server
+systemctl unmask redis-server || true
```

Comment:
maybe a small comment that this ideally will be updated in the future with an environment variable that will help determine if this is done in a running environment or chrooted dev building time.